From f96ec0cf13763a7499c89a320af2fd036b9f24c9 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Thu, 15 Jul 2010 13:09:20 +0800
Subject: [PATCH 118/280] ENGR00125135 vpu: fix code error for getting picType
 in encoder

fix code error for getting picType and skipEncoded in encoder

Signed-off-by: Sammy He <r62914@freescale.com>
---
 vpu/vpu_lib.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/vpu/vpu_lib.c b/vpu/vpu_lib.c
index df27c3b..771cf93 100644
--- a/vpu/vpu_lib.c
+++ b/vpu/vpu_lib.c
@@ -1207,8 +1207,8 @@ RetCode vpu_EncGetOutputInfo(EncHandle handle, EncOutputInfo * info)
 	IOClkGateSet(true);
 
 	val = VpuReadReg(RET_ENC_PIC_TYPE);
-	info->skipEncoded = (val >> 2) && 0x01;
-	info->picType = val && 0x03;
+	info->skipEncoded = (val >> 2) & 0x01;
+	info->picType = val & 0x03;
 
 	if (pEncInfo->ringBufferEnable == 0) {
 		if (pEncInfo->dynamicAllocEnable == 1) {
-- 
1.8.0

